test-app/package.json

```diff
@@ -12,72 +12,73 @@
+    "@babel/core": "^7.25.2",
+    "@babel/eslint-parser": "^7.25.1",
+    "@babel/plugin-proposal-decorators": "^7.24.7",
     "@babel/plugin-proposal-nullish-coalescing-operator": "^7.10.4",
     "@babel/plugin-proposal-optional-chaining": "^7.6.0",
-    "@ember/optional-features": "^2.0.0",
-    "@ember/string": "^3.1.1",
-    "@ember/test-helpers": "^2.9.3",
-    "@embroider/compat": "^3.2.3",
-    "@embroider/core": "^3.3.0",
+    "@ember/optional-features": "^2.1.0",
+    "@ember/string": "^4.0.0",
+    "@ember/test-helpers": "^3.3.1",
+    "@embroider/compat": "^3.8.3",
+    "@embroider/core": "^3.5.2",
     "@embroider/test-setup": "^2.1.1",
     "@embroider/webpack": "^3.2.0",
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
     "@types/ember": "^3.1.1",
     "broccoli-asset-rev": "^3.0.0",
-    "concurrently": "^8.0.1",
-    "ember-auto-import": "^2.6.3",
-    "ember-cli": "~4.12.2",
-    "ember-cli-babel": "^7.26.11",
-    "ember-cli-dependency-checker": "^3.3.1",
+    "concurrently": "^8.2.2",
+    "ember-auto-import": "^2.8.1",
+    "ember-cli": "~5.12.0",
+    "ember-cli-babel": "^8.2.0",
+    "ember-cli-clean-css": "^3.0.0",
+    "ember-cli-dependency-checker": "^3.3.2",
     "ember-cli-flash": "workspace:*",
     "ember-cli-github-pages": "0.2.2",
-    "ember-cli-htmlbars": "^6.2.0",
+    "ember-cli-htmlbars": "^6.3.0",
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
     "ember-load-initializers": "^2.1.2",
-    "ember-qunit": "^8.0.2",
-    "ember-resolver": "^10.0.0",
-    "ember-source": "~4.12.0",
+    "ember-qunit": "^8.1.0",
+    "ember-resolver": "^13.1.0",
+    "ember-source": "~5.12.0",
     "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^5.7.2",
+    "ember-template-lint": "^6.0.0",
     "ember-try": "^2.0.0",
-    "eslint": "^8.37.0",
-    "eslint-config-prettier": "^8.8.0",
-    "eslint-plugin-ember": "^11.5.0",
-    "eslint-plugin-n": "^15.7.0",
-    "eslint-plugin-prettier": "^4.2.1",
-    "eslint-plugin-qunit": "^7.3.4",
+    "eslint": "^8.57.1",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-ember": "^12.2.1",
+    "eslint-plugin-n": "^16.6.2",
+    "eslint-plugin-prettier": "^5.2.1",
+    "eslint-plugin-qunit": "^8.1.2",
     "lerna-changelog": "^1.0.1",
     "loader.js": "^4.7.0",
-    "prettier": "^2.8.7",
-    "qunit": "^2.19.4",
-    "qunit-dom": "^2.0.0",
-    "stylelint": "^15.4.0",
-    "stylelint-config-standard": "^32.0.0",
-    "stylelint-prettier": "^3.0.0",
-    "webpack": "^5.88.2"
+    "prettier": "^3.3.3",
+    "qunit": "^2.22.0",
+    "qunit-dom": "^3.2.1",
+    "stylelint": "^15.11.0",
+    "stylelint-config-standard": "^34.0.0",
+    "stylelint-prettier": "^4.1.0",
+    "webpack": "^5.95.0"
   },
   "peerDependencies": {
     "ember-source": "^4.0.0"
   },
   "engines": {
-    "node": "14.* || 16.* || >= 18"
+    "node": ">= 18"
```

Comment:
I guess this could be considered a breaking change? I'm not sure which version this addon officially supported before, but CI seems to run against node 16.